src/io/filesystem/zip_filesystem.h

```diff
@@ -48,8 +48,8 @@ class ZipFilesystem : public FileSystem {
 	StreamRead* open_stream_read(const std::string& fname) override;
 	StreamWrite* open_stream_write(const std::string& fname) override;
 
-	FileSystem* make_sub_file_system(const std::string& fs_dirname) override;
-	FileSystem* create_sub_file_system(const std::string& fs_dirname, Type) override;
+	FileSystem* make_sub_file_system(const std::string& path) override;
```

Comment:
I searched for this and found another declaration in layered filesystem which might need a fix